slicerio/segmentation.py

```diff
@@ -298,15 +298,19 @@ def read_segmentation(filename, skip_voxels=False):
                 spaceToLps = np.diag([-1.0, -1.0, 1.0, 1.0])
             else:
                 # LPS and RAS are the most commonly used image orientations, for now we only support these
-                raise IOError("space field must be 'left-posterior-superior' or 'right-anterior-superior'")
+                raise IOError(
+                    "space field must be 'left-posterior-superior' or 'right-anterior-superior'"
+                )
             continue
-        elif header_key == 'kinds':
-            if header[header_key] == ['domain', 'domain', 'domain']:
-                multiple_layers = False
-            elif header[header_key] == ['list', 'domain', 'domain', 'domain']:
-                multiple_layers = True
+        elif header_key == "kinds":
+            if header[header_key] == ["domain", "domain", "domain"]:
+                multiple_layers = False  # noqa: F841
```

Comment:
If we don't use multiple_layers then it is probably better to convert this to a comment than adding noqa

Reply:
done